app/scripts/models/account.js

```diff
@@ -395,6 +395,8 @@ define(function (require, exports, module) {
         var email = this.get('email');
         var sessionToken = this.get('sessionToken');
 
+        this._metrics.logEvent('core.signin.attempt');
```

Comment:
How are these different from https://github.com/mozilla/fxa-content-server/blob/08ff83805b36d6336babd55cbe6f69db4540f6c4/app/scripts/views/mixins/signin-mixin.js#L37-L40 ? I think those flow event should also be logged already